backend/user_management/main.py

```
@@ -0,0 +1,89 @@
+
+from fastapi import FastAPI
+import uvicorn
+import dotenv
+
+from fastapi.security import OAuth2PasswordBearer
+from fastapi import FastAPI, Depends, HTTPException, Header, APIRouter, BackgroundTasks, status
```

Ruff (0.8.2)

Redefinition of unused `FastAPI` from line 2

Remove definition: `FastAPI`

(F811)